lib/Response.php

```diff
@@ -103,6 +103,14 @@ public function flush();
     public function end(string $finalBodyChunk = ""): \Amp\Promise;
 
     /**
+     * Signal an extraordinary response end.
+     *
+     * It succeeds an eventual pending request body message and will reset the
+     * active HTTP stream.
```

Comment:
Why does it succeed a pending request body message instead of failing it?

Reply:
It actually fails it, the docstring is wrong.